Entities/AHuman.cpp

```diff
@@ -768,21 +768,20 @@ bool AHuman::OnSink(const Vector &pos)
 
 bool AHuman::AddPieMenuSlices(PieMenuGUI *pPieMenu)
 {
-    if (m_pItemInReach) {
-		PieSlice pickUpSlice(m_pFGArm ? "Pick Up " + m_pItemInReach->GetPresetName() : "NO ARM!", PieSlice::PieSliceIndex::PSI_PICKUP, PieSlice::SliceDirection::UP, m_pFGArm && m_pFGArm->IsAttached());
-		
-        pPieMenu->AddSlice(pickUpSlice);
-    } else {
-        PieSlice reloadSlice(m_pFGArm ? "Reload" : "NO ARM!", PieSlice::PieSliceIndex::PSI_RELOAD, PieSlice::SliceDirection::UP, (m_pFGArm && m_pFGArm->IsAttached() && m_pFGArm->GetHeldDevice() && !m_pFGArm->GetHeldDevice()->IsFull()) || (m_pBGArm && m_pBGArm->IsAttached() && m_pBGArm->GetHeldDevice() && !m_pBGArm->GetHeldDevice()->IsFull()));
-        pPieMenu->AddSlice(reloadSlice);
+	if (m_pItemInReach) {
+		PieSlice pickUpSlice(m_pFGArm ? "Pick Up " + m_pItemInReach->GetPresetName() : "NO ARM!", PieSlice::PieSliceIndex::PSI_PICKUP, PieSlice::SliceDirection::UP, m_pFGArm && m_Status != INACTIVE);
```

Comment:
Opening the pie menu is one of the things that's not disabled when `Status == INACTIVE`, otherwise you couldn't access crucial things like the buy menu (at least that's how I feel).

Reply:
Okay, I didn't look into it but I guess you can still select an actor (with actor switch buttons) when it's inactive?
If so then yeah, makes sense you'd want to have the pie menu usable and correct.

Reply:
Yeah, you can still select it. Not sure if it would be better if you couldn't, but at least it's clear to the player that the actor is still alive. Like I said, this status is widely unused - could deserve an issue of its own where we decide what exactly it should encompass.

Reply:
Okay cool.
And yeah, I'd say that's future worries. Actor and AHuman and such are still mostly ugly old code (I always hate working with them or reviewing changes to them cause they're such disgusting messes lol) so all this stuff may be reworked in future anyway.